src/cmd/operator/webhook.go

```diff
@@ -51,11 +52,11 @@ func setupWebhookServer(ns string, cfg *rest.Config) (manager.Manager, func(), e
 		waitForCertificates(newCertificateWatcher(mgr, ns, webhook.SecretCertsName))
 	}
 
-	if err := mutation.AddNamespaceMutationWebhookToManager(mgr, ns); err != nil {
+	if err := namespace.AddNamespaceMutationWebhookToManager(mgr, ns); err != nil {
```

Comment:
`namespace` and `pod` don't really feel like great package names, because it looks like a variable.

Maybe `namespace_mutation`?